frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -75,11 +76,12 @@ export const createDeviceSet = (
   osdSize: string,
   portable: boolean,
   resources?: ResourceConstraints,
+  arbiter?: boolean,
```

Comment:
Instead pass `replica` field. we would keep it generic and allow the params to be of device set type only.
This will scale well for future additions as well.